src/libtriton/arch/arm/aarch64/aarch64Semantics.cpp

```diff
@@ -3586,6 +3586,13 @@ namespace triton {
 
           /* Spread taint */
           expr->isTainted = this->taintEngine->setTaint(dst, this->taintEngine->isTainted(src1) | this->taintEngine->isTainted(src2));
+
+          /* Set condition flag */
+          if (this->astCtxt->bvlshr(op1, op2)->evaluate() != 0)
```

Comment:
I think it's more something like this:
`this->astCtxt->extract(0, 0, this->astCtxt->bvlshr(op1, op2))->evaluate()` ?